LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2015 President and Fellows of Harvard College
+Copyright (C) 2016 Python LTI Initiative
```

Comment:
Dropping the copyright statement makes sense, but is that acceptable to do? The one you linked has "and others", rather than leaving it out altogether.